iron-swipeable-container.html

```diff
@@ -0,0 +1,266 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<!--
+`<iron-swipeable-container>` is a container that allows any of its nested
+children (native or custom elements) to be swiped away. By default it supports
+a curved or horizontal transition, but the transition duration and properties
+can be customized.
+
+Example:
+
+    <iron-swipeable-container>
+      <div>I can be swiped</div>
+      <paper-card heading="Me too!"></paper-card>
+    </iron-swipeable-container>
+
+To disable swiping on individual children, you must give them the `.disable-swipe`
+class. Alternatively, to disable swiping on the whole container, you can use its
+`disable-swipe` attribute:
+
+    <iron-swipeable-container>
+      <div class="disable-swipe">I cannot be swiped be swiped</div>
+      <paper-card heading="But I can!"></paper-card>
+    </iron-swipeable-container>
+
+    <iron-swipeable-container disable-swipe>
+      <div>I cannot be swiped</div>
+      <paper-card heading="Me neither :("></paper-card>
+    </iron-swipeable-container>
+
+It is a good idea to disable text selection on any of the children that you
```

Comment:
In Polytrack, I attempted to do this by measuring how much time there was between `down` and the first `track` event to decide if the user intended to drag. If the time is < 50ms, I assume the user intended to drag, otherwise I assume that there was some other desired intention.